fe/fe-core/src/main/java/com/starrocks/ha/BDBHA.java

```diff
@@ -77,8 +82,7 @@ public boolean fencing() {
 
         for (int i = 0; i < RETRY_TIME; i++) {
             try {
-                long count = epochDb.getDb().count();
-                long myEpoch = count + 1;
+                long myEpoch = getLargestEpoch(epochDb.getDb()) + 1;
```

Comment:
Is it safe to do so?

Reply:
Yes, this process is before all write ops.